core/src/migrations/000085-addRecordsPropertyInvalid.ts

```diff
@@ -0,0 +1,18 @@
+import Sequelize from "sequelize";
+
+export default {
+  up: async (
+    queryInterface: Sequelize.QueryInterface,
+    DataTypes: typeof Sequelize
+  ) => {
+    await queryInterface.addColumn("records", "invalid", {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      defaultValue: false,
+    });
```

Comment:
This migration would 'suppress' any future failures to set `invalid` by converting them into "false" at the DB level.  Usually, we do a 2-phase migration that 1) adds the column and updates the old values and 2) changes the column to be non-null without a default so we get warnings if the code forgets:

```ts
await queryInterface.addColumn("records", "invalid", {
      type: DataTypes.BOOLEAN,
      allowNull: false,
      defaultValue: false,
    });

await queryInterface.updateColumn("records", "invalid", {
      type: DataTypes.BOOLEAN,
      allowNull: false,
    });
```